src/signatureUtils.js

```diff
@@ -33,12 +34,17 @@ const ADOBE_IOEVENTS_DOMAIN = 'https://adobeioevents.com'
 async function verifyDigitalSignature (signatureOptions, recipientClientId, signedPayload) {
   const signatures = [signatureOptions.digiSignature1, signatureOptions.digiSignature2]
   // complete public key url is the concatenation of the fixed adobe ioevents domain and the relative path of key
-  // url format for prod is - https://adobeioevents.com/prod/keys/pub-key-<random-uuid>.pem
-  // url format for stage is - https://adobeioevents.com/stage/keys/pub-key-<random-uuid>.pem
-  const pubKeyUrl1 = ADOBE_IOEVENTS_DOMAIN + signatureOptions.publicKeyPath1
-  const pubKeyUrl2 = ADOBE_IOEVENTS_DOMAIN + signatureOptions.publicKeyPath2
-  const keys = await fetchPemEncodedPublicKeys(pubKeyUrl1, pubKeyUrl2)
-  return await verifySignature(signatures, signedPayload, keys, recipientClientId)
+  // example url format - https://adobeioevents.com/prod/keys/pub-key-<random-uuid>.pem
+  const pubKeyUrl1 = ADOBE_IOEVENTS_SECURITY_DOMAIN + signatureOptions.publicKeyPath1
+  const pubKeyUrl2 = ADOBE_IOEVENTS_SECURITY_DOMAIN + signatureOptions.publicKeyPath2
+  /* istanbul ignore else */
```

Comment:
Why do we this `instanbul ignore else` comment? It is there at other places as well.

Reply:
This is used to skip the else clause from code coverage

Reply:
It is there at few other places where coverage needed to be skipped.